datalad_catalog/webcatalog.py

```diff
@@ -93,12 +143,14 @@ def create(self, force=False):
         """
         Create new catalog directory with assets (JS, CSS), artwork, config, the main html, and html templates
         """
-
         # TODO: first validate config file (using jsonschema?)
         # Check logo path, if added to config
-        if cnst.LOGO_PATH in self.config and self.config[cnst.LOGO_PATH]:
-            if not Path(self.config[cnst.LOGO_PATH]).exists():
-                msg = f"Error in config: the specified logo does not exist at path: {self.config[cnst.LOGO_PATH]}"
+        if (
+            cnst.LOGO_PATH in self.catalog_config
+            and self.catalog_config[cnst.LOGO_PATH]
+        ):
+            if not Path(self.catalog_config[cnst.LOGO_PATH]).exists():
+                msg = f"Error in config: the specified logo does not exist at path: {self.catalog_config[cnst.LOGO_PATH]}"
                 raise FileNotFoundError(msg)
```

Comment:
Probably the conditionals can be simplified as:
```suggestion
        if (
            self.catalog_config.get(cnst.LOGO_PATH) is not None
            and not Path(self.catalog_config[cnst.LOGO_PATH]).exists():
        ):
            msg = f"Error in config: the specified logo does not exist at path: {self.catalog_config[cnst.LOGO_PATH]}"
            raise FileNotFoundError(msg)
```